<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CadastroUsuario</class>
 <widget class="QDialog" name="CadastroUsuario">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>616</width>
    <height>269</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cadastrar Usuário</string>
  </property>
  <property name="locale">
   <locale language="Portuguese" country="Brazil"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1" rowspan="2">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="pushButtonCadastrar">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>Cadastrar</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonAtualizar">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>Atualizar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonBuscar">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../qrs/resources.qrc">
         <normaloff>:/search.png</normaloff>:/search.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonNovoCad">
       <property name="text">
        <string>Novo Cadastro</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonRemover">
       <property name="text">
        <string>Remover</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCancelar">
       <property name="text">
        <string>Cancelar</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Cadastro Usuário</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QLabel" name="labelNome">
             <property name="text">
              <string>Nome completo</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="lineEditNome"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="labelLoja">
             <property name="text">
              <string>Loja</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="ComboBox" name="comboBoxLoja"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="labelTipo">
             <property name="text">
              <string>Tipo</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QComboBox" name="comboBoxTipo">
             <item>
              <property name="text">
               <string>VENDEDOR</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>GERENTE</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>ADMINISTRADOR</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="labelUser">
             <property name="text">
              <string>Nome de Usuário</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLineEdit" name="lineEditUser"/>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="labelSigla">
             <property name="text">
              <string>SIGLA</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QLineEdit" name="lineEditSigla"/>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="labelPasswd">
             <property name="text">
              <string>Senha</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QLineEdit" name="lineEditPasswd">
             <property name="echoMode">
              <enum>QLineEdit::Password</enum>
             </property>
            </widget>
           </item>
           <item row="6" column="0">
            <widget class="QLabel" name="labelPasswd_2">
             <property name="text">
              <string>Conf. de Senha</string>
             </property>
            </widget>
           </item>
           <item row="6" column="1">
            <widget class="QLineEdit" name="lineEditPasswd_2">
             <property name="echoMode">
              <enum>QLineEdit::Password</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Permissões</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <widget class="QTableWidget" name="tablePermissoes">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <row>
          <property name="text">
           <string>Produtos</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>Clientes</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>Usuários</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>Profissionais</string>
          </property>
         </row>
         <column>
          <property name="text">
           <string>Leit.</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Grav.</string>
          </property>
         </column>
         <item row="0" column="0">
          <property name="text">
           <string/>
          </property>
         </item>
         <item row="0" column="1">
          <property name="text">
           <string/>
          </property>
         </item>
         <item row="1" column="0">
          <property name="text">
           <string/>
          </property>
         </item>
         <item row="1" column="1">
          <property name="text">
           <string/>
          </property>
         </item>
         <item row="2" column="0">
          <property name="text">
           <string/>
          </property>
         </item>
         <item row="2" column="1">
          <property name="text">
           <string/>
          </property>
         </item>
         <item row="3" column="0">
          <property name="text">
           <string/>
          </property>
         </item>
         <item row="3" column="1">
          <property name="text">
           <string/>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ComboBox</class>
   <extends>QComboBox</extends>
   <header>src/combobox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>lineEditNome</tabstop>
  <tabstop>comboBoxLoja</tabstop>
  <tabstop>comboBoxTipo</tabstop>
  <tabstop>lineEditUser</tabstop>
  <tabstop>lineEditSigla</tabstop>
  <tabstop>lineEditPasswd</tabstop>
  <tabstop>lineEditPasswd_2</tabstop>
  <tabstop>pushButtonCadastrar</tabstop>
  <tabstop>pushButtonAtualizar</tabstop>
  <tabstop>pushButtonNovoCad</tabstop>
  <tabstop>pushButtonRemover</tabstop>
  <tabstop>pushButtonCancelar</tabstop>
 </tabstops>
 <resources>
  <include location="../qrs/resources.qrc"/>
 </resources>
 <connections/>
</ui>
